include/couch_replicator_api_wrap.hrl

```diff
@@ -22,10 +21,9 @@
     timeout,            % milliseconds
     ibrowse_options = [],
     retries = 10,
-    wait = 250,         % milliseconds
+    wait = [250, 25],  % [normal error wait time, 429 backoff wait time]
```

Comment:
agree it should be a tuple, or a record. list implies there's more here than two independent values.